WcaOnRails/app/views/competitions/mycomps.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide(:title, 'Upcoming competitions') %>
+
+<div class="container">
+  <h2><%= yield(:title) %></h2>
+
+  <table class="table">
+    <tr>
+      <th>Name</th>
+      <th>Location</th>
+      <th>Start date</th>
+      <th>Delegate(s)</th>
+      <th>Organizer(s)</th>
+      <th></th>
+    </tr>
+    <% @competitions.each do |competition| %>
+      <% if competition.isConfirmed %>
+        <% if competition.showAtAll %>
+          <% message = "This competition is confirmed and visible." %>
+        <% else %>
+          <% message = "This competition is confirmed but not visible yet." %>
+        <% end %>
+      <% else %>
+        <% if competition.showAtAll %>
+          <% message = "This competition is not confirmed but is visible." %>
+        <% else %>
+          <% message = "This competition is not confirmed and not visible." %>
+        <% end %>
+      <% end %>
+      <tr class="<%= competition.isConfirmed ? "success" : "danger" %> <%= competition.showAtAll ? "visible" : "not-visible" %>" data-toggle="tooltip" data-placement="top" data-container="body" title="<%= message %>">
+        <td><%= link_to competition.name, "/results/c.php?i=" + competition.id %></td>
```

Comment:
Can we use `competition_path(competition)` here instead? That way we won't have to change this once we're done with php.